src/pages/_app.tsx

```diff
@@ -29,6 +29,13 @@ export default function App({
   const [trpcClient] = useState(() =>
     trpc.createClient({
       links: [
+        // Log in development and only log errors in production
+        loggerLink({
+          enabled: (opts) =>
+            (process.env.NODE_ENV === "development" &&
+              typeof window !== "undefined") ||
+            (opts.direction === "down" && opts.result instanceof Error),
```

Comment:
ideally this would use the existing logger we have defined in `backed/lib/logger.ts`, I think we should be able to pass the export of that to a `console` option here?

https://trpc.io/docs/client/links/loggerLink